_security/authentication-backends/saml.md

```diff
@@ -224,7 +224,7 @@ SAML, unlike other protocols, is not meant to be used for exchanging user creden
 
 Name | Description
 :--- | :---
-`exchange_key` | The key to sign the token. The algorithm is HMAC256, so it should have at least 32 characters.
+`exchange_key` | The key to sign the token. The algorithm is HMAC-SHA512, so it should have at least 64 characters, and base64 URL encoding.
```

Comment:
We need people to base64 encode the token before putting it into the config I believe right? Can we make that explicit by splitting this line into two sentences and add this detail?

Reply:
Just checking on a few more things with this @scrawfor99 - I might have another piece of info that has to be added also.

Reply:
Just have to do some more testing on this one, because it might be the case that HMAC256 is actually allowed, and it's only if the user doesn't specify a value for the exchange_key that it defaults to a HMAC-SHA512 code with 64 bytes.